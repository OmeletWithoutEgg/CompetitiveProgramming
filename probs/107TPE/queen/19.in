1258
135758784 -101002778
35909348 -98250957
67228461 -143638256
-55356065 -54345907
59326497 -139609336
121843223 -62498200
54991999 -57952062
-64130019 -44679044
127032569 -125681608
135074879 -106298094
-133138811 -116798572
-80188311 -35547281
53155992 -59467994
106544355 -51716272
-64546408 -44330152
-133152394 -116783896
110299382 -53612916
-79347550 -35844859
-82536467 -34803825
106622621 -51751835
-46710293 -77734312
115418352 -56898796
-71089857 -39737644
-91696266 -132602443
-83428855 -131103657
53793782 -58924623
-96907253 -132826851
127732983 -124640410
-46706356 -87884986
-89757565 -33278861
52242754 -134231173
128693913 -71397201
-79395326 -129830387
-140442564 -106591516
-99344556 -132745090
56408353 -137631805
-78149071 -129359294
135353435 -89886175
-76766512 -36866372
-144894157 -70452764
135156021 -88665114
63459654 -52573401
-127497580 -43636446
-60892384 -47677108
-54052702 -56324909
-116105852 -128803391
-46460878 -81789070
102089743 -49948954
43121336 -123140152
120178791 -133654650
45821047 -127150327
128824694 -122895362
-69353100 -124849824
131668823 -117383779
-136585868 -53010873
133028958 -80557592
-65273236 -43739232
135629043 -102447969
-116503057 -37026349
55630684 -137052147
118308846 -59181120
87242697 -47273171
50455281 -132521429
79085833 -146788698
-143399161 -65630620
-130556951 -46267757
-139268034 -57009929
68035588 -50555198
-95149441 -32845861
101304372 -144822842
65787083 -51478564
-133531425 -116369553
128655775 -123176063
-110149989 -130915447
-146368856 -79980406
-119851512 -127014598
-92185888 -132646778
-71207907 -39668386
128299514 -70755447
131441682 -76617699
-143984739 -67322343
126094818 -67517424
129833049 -121126387
81073174 -147021652
129440668 -72675445
-142650090 -101948501
125773737 -127421988
-56199589 -53165845
125044001 -128363043
36769225 -106540756
37764420 -110783640
-90299233 -132449172
67203163 -143628065
47002946 -128672691
46220369 -127678341
131759519 -77333973
-46540410 -85833349
-104071037 -132244736
-146439791 -83842441
-80523816 -35433218
43472876 -70797849
113593649 -138884805
-146421277 -84525346
-120778977 -126510791
133173178 -80970382
-145260184 -93672012
66616969 -51122851
135455602 -90607877
-83182704 -131036588
-70369819 -40169583
128996792 -71905224
-78637844 -129548582
95051819 -48099924
133427883 -112780788
99582868 -49163924
-47340193 -73436361
135895680 -97867541
-144719218 -95870750
133862569 -111380894
45044305 -68430505
-68174840 -41591680
95260369 -48139212
74287282 -145888015
74103237 -48666547
-90600706 -33172270
123765070 -129907741
-141314039 -60755797
43397408 -70919254
-139342010 -57133009
48533655 -130478592
-144299325 -68322110
134712856 -86426943
102744217 -144332222
-140487336 -106508439
40874170 -118997385
36701951 -106177457
-80969796 -35285684
-91210298 -132553632
66019323 -51377229
-128090255 -121544549
69897362 -49884946
-141266799 -104997844
35941567 -99307415
135685158 -92631133
-99724836 -132721585
133352060 -81499238
66399943 -143296084
48976059 -130969582
-143038776 -64677031
-49185242 -66518313
-87638888 -33611433
-48509063 -97029663
-118659843 -38032450
135897111 -96773525
78768737 -146744045
105607138 -51302907
121362957 -132501944
-84258926 -131319932
-145527848 -73270121
-143554856 -99596128
-139239839 -56963229
-65478630 -122081588
60564776 -140361450
-134844217 -114858443
134031829 -110792931
132959966 -80364120
122004311 -131844684
-108108919 -131450656
135767896 -100879510
40369499 -117919716
49695106 -131740583
119951139 -133867706
-118154819 -37785589
36046559 -101088031
-50976901 -103618157
88752011 -147173686
-58865392 -49853307
135886777 -96130324
-108772566 -34371167
-127314405 -43492032
130791806 -119270452
-134771519 -114945387
88779073 -147172132
48154912 -64462575
-57218195 -51831192
-144598004 -69346513
88863436 -147167194
-115183181 -36470875
50386569 -132452236
134025169 -83693661
64159672 -142281593
51104684 -61347944
-48500512 -97000764
135477192 -90770868
-101935325 -33130733
58004410 -138750520
-61292541 -47276886
-46615538 -78764457
104963401 -51032132
-127621721 -43735108
-49169155 -66565003
-106378128 -33824518
75675235 -48311628
-67782944 -41863148
-75545634 -128249255
-136365139 -112941841
126409487 -67947526
-117226141 -37349775
89028184 -47353112
-61947721 -46640707
135793957 -100501079
-76555893 -36957142
36027953 -100837824
-136476966 -112793037
-128506239 -121200827
135892859 -96444501
107287193 -52060395
42700676 -72078398
96758298 -48448516
126828829 -68536066
37795304 -110893051
-104561765 -33491326
135739505 -93259253
134048491 -83776699
-143173721 -65027261
134780462 -86736345
57704220 -55963114
36127236 -92487664
135883441 -98519537
45670420 -67562924
89496057 -147125600
-138898703 -109250769
-109773539 -131021109
-48781194 -97917974
-59337841 -116335403
-124459416 -41410634
-65533864 -43532777
-59505780 -116520482
-117216743 -128312400
-99734442 -32936930
-135892972 -113557706
118407235 -59265091
135331523 -104769661
-99701980 -132723080
93122238 -146730685
-58341305 -50460399
-107896573 -131501084
127341159 -69280438
-91896805 -33036695
36544568 -105261220
58448619 -55464540
135899420 -97288268
-101286573 -132594473
-111057006 -130647737
-49529792 -65552132
-46744451 -77411186
124239708 -65161151
129244526 -72331187
-116105338 -36854273
107849545 -142179422
-48932925 -67268687
41385092 -120025141
64739468 -51953292
-46710208 -87922743
130465293 -74586117
-63876881 -120762934
-46540157 -85829145
-75403101 -37474500
-124130520 -41190110
-89088126 -33373892
-93107301 -132717076
37946413 -83095036
125422549 -66629595
-65403897 -43635376
-53591009 -57078185
119996056 -60684369
135595817 -91753383
-64695665 -44206950
71912392 -145258903
57479767 -138392975
-71347562 -126070871
47236810 -128960041
135289925 -105038380
81310399 -47466162
127260173 -69160837
-98646552 -132780673
-75619775 -37374579
119016987 -134714802
-87421508 -132007038
-146315110 -86500057
111615978 -140134714
135888902 -96229077
-65923914 -43229015
-71078024 -125913273
-100552834 -132660330
-49055947 -98759972
-53645495 -56987716
51013455 -133073677
-99113292 -32899920
74562501 -48557342
-144998752 -70869643
125356617 -66544696
45080370 -126130795
44198366 -124841738
-140160033 -58550585
59824262 -54592664
-47575162 -93376171
135899290 -97136580
103316002 -50386553
58172852 -55646990
-128465388 -121234918
-82865999 -130948306
111534424 -54326730
38834701 -80375812
135791282 -93968304
82329196 -47382753
117479607 -58490547
-62485914 -119522763
-104764532 -132132791
-132543871 -48227828
-100068565 -32960049
97223206 -48554282
-110655936 -34889467
-131777947 -47446168
-138232364 -55365523
-146202320 -77857724
-134410719 -50286836
-78595816 -36125346
73812599 -48738029
130851394 -119148525
-47203420 -91475794
-144749167 -69898487
-139058331 -108992572
71721144 -49307484
102080044 -144564614
75663271 -146196120
52560186 -59992613
-68357001 -124190516
-79053083 -35953094
-143946043 -67204220
95438751 -146336703
134004178 -110890858
44193078 -124833743
40679196 -75921340
-94644962 -32861536
135123158 -106031493
60129127 -140102429
-145965493 -75884715
115806561 -137324608
-60877066 -47692609
-71310649 -39608462
106730580 -51801157
-84120634 -34372929
112915151 -139328285
62955493 -141680069
-145408465 -72676357
103793945 -50566942
135842021 -99650460
-105584658 -33670431
-146445244 -83523142
-113068073 -35671315
95926031 -48270767
-46632728 -78558920
-46815701 -88864677
-54643671 -110255671
91237827 -146969323
-133541574 -49299554
-88479902 -33468264
110112072 -141001530
115736270 -57133273
52266080 -60257858
-142121930 -62479972
135715979 -101534312
-55584659 -111638986
101583782 -144731446
133530412 -112463334
37142003 -108332725
-46511259 -85301959
-144213805 -68042970
50154768 -132216805
-51716650 -105165498
55248292 -57751863
83280116 -47323780
-141487416 -104546143
116333269 -136926039
-123684309 -124760980
-47429527 -72980742
-61123479 -118213012
76519109 -146367339
-47827888 -71172011
-116694109 -128547413
-146117142 -88589260
-137403812 -111513266
-133594955 -116299183
-103669230 -33352850
-48143891 -69924922
-139933070 -107511489
-140201867 -58626056
104809044 -50968771
114383253 -56161671
-51044718 -103765861
52131971 -134130030
125455501 -66672166
-145454742 -92755749
112148413 -139810863
124403715 -129152147
-112643059 -130134194
120270475 -133568088
-66360087 -122761301
-46495945 -84970437
40064564 -117234239
-73634213 -127319805
98240088 -48801970
-50814839 -62397929
40445220 -118085746
41547683 -74170010
-126453341 -122826483
-101947153 -132525843
125091472 -66207035
-75958313 -128436925
78683360 -146731668
135325870 -89703498
119473767 -60204295
86523612 -47259021
65067179 -51801662
56421718 -56868686
116073974 -57386629
-46481593 -81053565
-59751833 -116788327
134878263 -107308678
135736631 -93223572
-46480306 -84567727
60788091 -140491922
-96467945 -132828938
35965967 -94676524
-93946065 -132766202
-146431706 -81474183
101783380 -144665041
-135606702 -51736215
-87820614 -132078638
-54828426 -110535246
-146087440 -76818044
-68578186 -41317998
-130978731 -46665824
-134579753 -115172825
-88257387 -33504708
51578651 -133615599
-139033824 -109032440
134712696 -108084025
-46601370 -86715809
-117620698 -128125787
36794916 -87834578
81257967 -47471022
-100967643 -33033445
82587356 -47364943
-146297962 -86725987
-63197656 -45488918
52312975 -60215281
-50754637 -62532933
-54746113 -110411192
-108925547 -131247557
-47602073 -93500112
-121594370 -39611308
-101850149 -33121406
-107384891 -34039295
-134894857 -114797642
-134062298 -49884749
-95871543 -132825595
-102381680 -33182029
-51972279 -105670279
-53022740 -107609329
-127230773 -43426556
121656429 -132204188
-117053181 -128386724
-75319818 -37513241
-121021160 -39282860
-91666673 -33058276
68677285 -144195488
51515486 -133555873
-122602546 -40213831
-130986008 -46672773
135341254 -89804882
-131463325 -118523080
-143267364 -100382868
-127264021 -43452552
-145381167 -93112283
132624777 -115052371
-65126844 -43856440
87770387 -47290142
-77630226 -36506013
35919655 -95833141
78109622 -146644586
135837128 -99750396
-68448068 -41405660
126504403 -68079172
119385588 -134385667
-57358629 -51654276
-124295760 -124357452
135893238 -98042055
134029931 -83710572
125524793 -66762000
120162364 -60840136
-117296687 -128275815
-90916691 -33136067
-69375385 -40793697
129223387 -122215789
118793191 -134911472
-140837872 -105844642
54368365 -136059659
85838713 -47255161
-145081203 -71209463
-102717513 -132434577
-64705382 -44198964
75574493 -146177468
-103624251 -33346308
92412145 -146829156
-57545444 -51421449
103313815 -50385740
-48944047 -67234765
-135199051 -114428244
-49336904 -66085280
-66670001 -42664929
135060706 -88135421
133414595 -112821407
-117760312 -37597610
36642180 -105841348
-52377632 -59214946
-145829243 -90683665
67000874 -143545993
-132470270 -117506668
-46450617 -82593760
53484095 -59186161
-119047733 -127431019
-109238230 -34491970
-57518743 -114203330
123728469 -129950137
88357131 -147194685
-47627981 -72039861
36177222 -102518393
100726540 -145006112
135062378 -88144439
-144914730 -95124309
45937090 -127305272
-82252840 -34886905
-112217980 -130277575
-144484823 -68948671
-98217179 -32860178
-64589947 -44294113
-135417953 -51499983
46306274 -66720194
-47002944 -90243953
135805059 -100325661
-116448224 -128655501
62552570 -141469648
38059287 -82718250
84984339 -47263499
-139710468 -107899590
-52359445 -59248921
-106175019 -33783807
64334257 -142365470
-143437801 -65736472
134969660 -87657594
-46450206 -82709681
-91683124 -33056697
46348791 -127845105
103857157 -50591219
123381115 -64162477
63578130 -52514097
130939268 -118967172
-54669922 -55362239
-140273818 -106901301
41044895 -119347445
-62995384 -119987849
124681100 -65695940
121130233 -61775686
-46861722 -76426097
-137120737 -53744645
134990973 -87767212
-145563077 -92205075
-99854351 -132712915
-81647074 -130587412
-118849853 -127530725
107971481 -52390611
92580254 -47703469
-114676187 -129388671
119358352 -60100036
-93949656 -132766382
70281229 -49757001
95689023 -48222851
135194984 -88891611
133165264 -113562821
78333936 -47830806
62712885 -52956320
-100579722 -132658109
69437932 -144467613
36345277 -103917396
37581167 -110113796
-47353033 -73369468
-64721644 -121472278
49680388 -131725125
134377561 -85012948
135874873 -95688247
-87465389 -132015074
133790653 -82887424
36563644 -89132305
54084088 -135826928
96152398 -48317647
-46463543 -81668017
-75751004 -37314670
135317943 -89651797
130698752 -119459195
-138967077 -56517279
115109608 -137835484
-135008249 -50997097
124057928 -64945233
-80446879 -35459140
35904645 -96533035
-46458697 -81899818
135682761 -101904858
-81717720 -130609252
135429850 -90418638
46693835 -66224151
85070369 -147248251
-62523509 -46100357
59720434 -54656296
130489756 -119875974
-51601663 -104933713
103574955 -144026653
97931538 -145785820
-145986344 -89622830
-53457051 -108355415
-77267755 -129002950
39901600 -116856134
-51719212 -105170626
-82710979 -34753609
-129390722 -120444271
132272413 -78558961
-115312343 -36523279
-77118844 -128940805
-53338482 -108154651
59738656 -54645104
37872948 -111164007
36047779 -101103864
83812116 -47298712
53973983 -135735844
131563538 -117621502
-98814671 -32884886
-118549283 -37977804
82124622 -47397820
132093304 -116389546
-100014684 -132701714
107477895 -142359114
96162900 -146190400
-135261310 -51306138
120001220 -60689185
-49803896 -64825190
-132537173 -48220842
36832454 -106869270
122315232 -131517194
59238194 -54956443
-130206423 -119714009
-53603736 -57057014
-69346960 -40812019
103607382 -144014385
124534888 -128993136
37096612 -108131009
-144377470 -68582409
100851393 -49543084
135898463 -97566240
-46863463 -76412634
-64704532 -121458221
-142134436 -103149820
88519764 -147186416
-80602958 -35406697
66013906 -143130673
37210153 -108628527
60268019 -140185658
-111433659 -35126826
-99132069 -32900925
69526964 -50011687
77693633 -47933072
48997470 -63517233
80893429 -147003893
36505069 -89496449
63252275 -52678154
-66949513 -42459181
-92688987 -132687284
63523318 -52541485
41370375 -119996347
-114806322 -36320364
132979482 -80418595
-81199551 -130446395
-89416905 -132331817
-84074292 -34384744
56621311 -137786492
135071378 -106317112
-68990815 -41043900
-111048609 -130650301
-52101328 -105919841
-83405524 -34560525
-46592139 -86595557
-117643784 -37542893
-135158399 -114478029
-132938916 -117013202
-110190485 -34753992
44069253 -69864683
36688615 -88406641
-52010566 -105744679
-74209402 -127610109
-110176806 -34750084
134722348 -86469822
77608786 -47947263
-72011291 -126449423
111237313 -140359552
-94818154 -32855580
64886353 -51884974
-47547642 -93247827
37346471 -109197072
-49251991 -66326157
-97719204 -32845051
120560307 -133291546
-73946257 -127478451
-135884849 -113568130
-136622017 -112598291
-82837940 -130940376
-122836149 -125299808
76081473 -48228522
-130372139 -46096221
-87052319 -131937825
-134027355 -49844898
120350369 -61017950
-73415333 -127206877
37407240 -85068761
83283284 -147186635
91802717 -47604835
-60522244 -48055506
92361780 -146835746
-60061994 -117120462
125827561 -67159537
-54835573 -110545979
107795344 -142205864
134544364 -108816717
135339755 -89794944
57555227 -138445004
38053744 -82736506
-93567678 -132745791
135807808 -100280658
48622739 -63931735
-72116071 -126507962
101708171 -144690172
-50588554 -62911062
75031283 -48450582
-122025135 -39864820
133805419 -82936737
-79776794 -129967059
57281151 -56255196
-111981232 -130355595
-59517870 -49124148
-141889289 -103692230
-66453899 -122831818
130060717 -73807720
-140731346 -106048934
-83807522 -131204205
-144994889 -94803912
115414157 -56895728
-48424433 -68917118
-104873967 -132114212
-146264473 -78524924
44849239 -125801258
-143247301 -100436284
97364522 -48587356
52567506 -59986065
59477543 -139703727
-85417790 -131596645
43129931 -123154351
-118051536 -37735967
36099889 -101727892
-46635431 -78527498
133579027 -82200022
91656913 -146922532
66542881 -51153888
91743554 -47597837
-102890942 -33245526
101688124 -49813401
-108287720 -131407434
-53735263 -108818282
38134904 -82471687
107475695 -142360166
-60450194 -117527780
135816481 -100134019
-106917253 -33936836
-98680521 -32878716
-51904332 -105537481
-136314511 -113008832
-81397633 -130509381
-133372776 -116544120
132908313 -80220896
80368707 -146948294
134526994 -85620690
98416387 -145663035
-52959350 -107497906
123748556 -64583368
-136852071 -112285313
95826475 -48250494
-81902224 -130665751
84612843 -47271680
121648070 -62297511
67418498 -143714302
-99210077 -132752707
-58516438 -115402551
36050903 -101144122
-122799070 -40335061
-139455825 -108333931
92519248 -47695282
-137231768 -53900538
-117454343 -37454717
-55864741 -53625661
-101183120 -132604419
91801856 -47604733
62134923 -53263669
-118807162 -38105792
84620258 -147238759
92541445 -47698252
119635700 -60351681
-95520358 -32837586
113683058 -138825187
-99820258 -132715230
-65396434 -122016594
76849934 -146429372
61410357 -53662863
-129270570 -120549153
-67172543 -123360742
58363597 -138989737
85108081 -47261387
101553163 -49768699
35974637 -94513779
135899317 -97361779
113220771 -139130468
-57982195 -114769872
39218755 -115169211
-142127834 -103164656
129335603 -72490251
69982200 -49856372
35910644 -96196232
102049527 -49935209
42200810 -121553901
-101843575 -132537191
110876829 -53940813
41838192 -120889997
-92437715 -32990191
-124652288 -124116162
-121627769 -39630757
115472368 -137571887
-61689432 -46888734
-135967624 -52196240
135726181 -101413843
122323052 -131508880
-134388359 -50260770
-97699336 -132813332
-141599062 -104313073
44956023 -68556041
-137501218 -54284192
-86704616 -33787874
79717884 -47638711
-69751475 -40553826
54981272 -57960500
45469175 -126672712
-76246134 -128565155
-131481261 -118505478
-111266954 -35074782
124321207 -129251486
104071541 -50674287
110797432 -140615124
-139830520 -57966600
-57082922 -113654713
-76593819 -128717170
-85932978 -131710338
-49087512 -98853571
-64426920 -44429485
-144361946 -68530289
-64518111 -121304264
79101386 -146790835
125667057 -66947768
-47283410 -91919604
99464433 -49130383
-133343784 -116575844
-81454674 -35130532
92417998 -146828386
135894067 -96522755
126896694 -125877220
124594189 -65589439
126746739 -68419428
122088726 -62753894
-130850350 -119114204
110228247 -53573221
129933000 -73568740
53686336 -135495372
107600206 -52209855
-143474686 -65838213
-119052941 -38229503
109048090 -141573739
-46471105 -81378509
81836758 -147089798
115705153 -57110151
-69552722 -40680003
-48810049 -98008829
106179245 -51552508
63998913 -52306628
117635970 -58618457
105623737 -143200219
36166016 -102411434
121468058 -132395881
-80074847 -130071425
130135203 -120561698
35900290 -96962038
42876960 -71778331
-75071761 -37629736
135852701 -95093918
-120139467 -126860886
113953333 -55866979
-73923531 -38190893
-56906692 -113428317
88769779 -47337581
-74401458 -127704982
94343042 -146537022
135495174 -103600375
-138592622 -109736278
-118612601 -127648824
134971201 -87665477
96938987 -48489069
-81918249 -130670622
56981120 -56466262
50614048 -132680282
69007186 -50195035
-88920906 -132258809
73141487 -145600082
48596705 -63960881
-86970051 -33735873
-141869515 -61922640
45509956 -126728678
-83783518 -34459957
134840619 -87019888
-70040034 -125284922
-111578862 -35172678
50285626 -132350095
64272007 -142335659
-48031665 -95305266
-121500210 -39556662
-143710312 -99152817
36189477 -91877343
76791976 -146418671
63198531 -52705499
-60764365 -117850805
72781613 -145503678
-95463830 -132819214
-89856888 -132392336
-97388812 -132820128
-55038465 -54809670
68947888 -50216362
118445553 -59297913
-89428353 -33324441
113847872 -55795691
-98723987 -32880676
93039360 -146742713
120264356 -60936370
-71529595 -39481851
134868369 -87153490
114805166 -56457349
-125719446 -123366618
132465619 -79045509
109839350 -141151369
113474130 -55546181
60853788 -53980238
-93027599 -132711671
-78615367 -36117876
-52369756 -106428232
-60811125 -117898388
125167744 -128206693
51994081 -134003285
92975650 -47758386
35978851 -94438075
-81392075 -130507626
135476475 -103744856
36473055 -104807169
-141218303 -60562268
35914133 -98467557
41021485 -75210397
35900389 -97564758
51522515 -60947719
120889232 -61537989
135481587 -103705683
117385515 -58414083
-62754287 -119769362
-46993976 -75473981
36357856 -104010299
39784945 -116580087
135140780 -105932076
-145898984 -90231934
95057929 -48101062
133048712 -113896796
-76544775 -36961965
-102235034 -33164728
-115512635 -36605372
-92392337 -132664018
73568218 -48799565
36181784 -102561310
-124428778 -41389932
102587942 -50122401
-50386208 -102274281
-138825872 -109367421
-47256002 -91770099
-57355661 -51657998
87133118 -147239902
121097795 -132766745
77514120 -146546975
110857542 -140580552
125998829 -127122134
93122374 -47779584
122876076 -130911145
125557533 -66804592
-103698248 -33357093
-75034113 -128010321
105481351 -143261085
135700709 -101708516
-49022961 -66996380
-47333088 -73473582
132889397 -80168783
100426321 -145098300
-146400362 -80600116
-76328897 -37056257
-64370606 -121181366
-46997567 -90208006
113418257 -139000953
-140615487 -106268553
41909184 -121021871
-80300363 -35508891
89807392 -47408080
-47871066 -94664525
112614945 -54990704
59343096 -54890504
-113733230 -35911012
119835776 -60535587
79076032 -146787349
107919328 -52364991
-136071576 -113327069
61551736 -53583741
54409022 -58417589
115557712 -137509147
132244788 -116019662
111699045 -54425463
133026362 -113959985
42632811 -72195203
-62131498 -46466377
-84205041 -131306353
135614189 -91921943
-95278332 -32842673
50365212 -62079575
-130776483 -119184092
134193399 -110204743
134834700 -86991627
86604815 -47260100
112860875 -55147162
-103284783 -33298278
41923909 -121049104
93872504 -47894917
66833196 -51033066
-46458154 -83728368
-114882778 -129307270
53990169 -135749267
-118074768 -127910780
-121739349 -125961899
-57325896 -51695366
132450163 -79006035
-55824792 -53681261
-84203268 -34351978
65998913 -51386078
135886927 -96136994
36596409 -105574502
56192131 -57037264
-146204684 -87776443
-136573646 -112663454
-49217903 -66423971
135897311 -96794749
126652590 -126223727
73203681 -145616451
134843007 -107478938
39311033 -115407845
-131222329 -118757893
56195283 -137475314
-61031422 -47537018
-94915249 -132805379
38754354 -113908706
-74538806 -127772204
41419115 -74418716
-63380152 -120330688
38232802 -112351234
-47039700 -75172836
97142664 -48535624
-146192622 -87896278
-52330983 -106355665
43742542 -124140004
120998617 -132864773
-111975893 -35300544
-112801935 -130079511
-56425070 -52862295
-46521694 -85504364
135889068 -96237204
-103965446 -33396978
-59512668 -49129850
84440088 -147233823
-76207544 -37109795
-108001072 -131476390
43384329 -123569876
133930955 -111146585
-129188539 -120620372
-126698533 -43016505
-60726190 -117811864
-125893314 -42417372
135429469 -90415875
125619316 -127625023
-120883708 -126452296
-58643549 -50107886
51254185 -133306573
127663263 -69763629
134696967 -108154684
-78166350 -129366086
100736757 -49507311
41798194 -120815281
133537156 -112442194
-74962072 -37681780
63748441 -52429533
-133801284 -49589111
-77426223 -129068465
107625413 -52222007
-93473270 -132740250
82724248 -47356044
-53612045 -57043205
-59774437 -48845145
83875838 -47296091
62801041 -52910290
-113428033 -35799711
112215085 -54740582
72768541 -145500122
120963900 -61611289
-58212574 -115045311
97550160 -48631462
-129995507 -119905932
-142446361 -102433549
-142207963 -62674174
135226708 -89080794
125948070 -127190161
41220343 -74810099
-146121940 -77110138
-58702565 -115618061
70762566 -144908827
-68682538 -41248122
71527250 -145145003
-102653201 -33215223
-54183980 -109541826
-96205251 -32829541
-50294723 -102056123
68130117 -50519148
128582369 -123296759
-100133833 -132693055
80649544 -146978748
-140369039 -106727134
-57308128 -113940176
55458434 -136920547
-146449145 -82525840
41276699 -74698266
-104505638 -132175752
-55814456 -53695671
-70657723 -125662984
-50203456 -63822330
-51712680 -60500338
-142833867 -101498242
-97081104 -32832924
-141811959 -61798048
86255065 -47256389
135851768 -99437785
-65953056 -122451316
-46497985 -85017526
88024132 -147209961
64131931 -142268189
-48993333 -98572471
-133673958 -116211297
-48205243 -95960478
86573056 -147250587
-123811213 -40979599
-49230017 -66389135
-88085042 -33533643
36311789 -103663386
118991117 -134737657
135055902 -106400687
-135913335 -113531551
-143091467 -64812853
-104533560 -33486697
-102998632 -33259634
66553300 -143360734
126680581 -126184309
-78664551 -129558756
-68889329 -124547114
-142018406 -103408693
-59132004 -49551886
-56834329 -52323310
-101830492 -132538609
-103394094 -33313485
-65786984 -122322935
-103095203 -132385395
-76240654 -37095149
112400067 -54855640
-48862008 -67486942
-132664163 -117304135
-134754676 -114965473
106593562 -142771637
43641573 -123981019
94241139 -146554373
36692767 -88383582
-121734231 -39692983
-100412083 -132671718
-47604623 -72146106
40904730 -75449691
-126635196 -42968463
103344828 -50397274
-73579544 -127291727
-48521217 -97070630
127209931 -69087012
90549220 -47471799
36314703 -90824356
109693979 -141230339
-104850051 -33539590
-52974672 -58132983
38955539 -114467651
135845022 -99587049
-146050574 -76521057
-125734439 -42302094
-122491982 -125511707
98928028 -145527846
96742964 -48445107
52053385 -60452336
-79137841 -35921725
39786701 -77925972
63352701 -52627274
-122815974 -40345547
-49373910 -99676366
-74354737 -37975886
133192586 -113483415
-50525869 -63056021
119279851 -60029495
86589003 -147250369
134707117 -108109146
38278862 -112495906
40482921 -76342432
82966594 -147169035
-71056547 -39757265
54550127 -136206648
-46846381 -89111827
-66759370 -42598819
