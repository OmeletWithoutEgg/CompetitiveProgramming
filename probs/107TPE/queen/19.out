-5275887.97 -90041476.32
