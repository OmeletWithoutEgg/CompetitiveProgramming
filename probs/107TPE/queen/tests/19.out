16252792.53 -59042721.70
